decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -21,6 +22,16 @@ def author_avatar_url
         author&.avatar&.url || ActionController::Base.helpers.asset_path("decidim/default-avatar.svg")
       end
 
+      # Public: Canpeople comment on this proposal?
+      #
+      # Until we have a way to store options fore features and its resources we
+      # assume all proposals can be commented.
+      #
+      # Returns Boolean
+      def commentable?
```

Comment:
I just hardcoded the values for `arguable` and `votable` options. We can include them as a extra configuration options in the future so we can call it like this:

```ruby
if @proposal.commentable?
  comments_for @proposal, arguable: @proposal.arguable_comments?, votable: @proposal.votable_comments?
end
```

I think we don't need to add this right now.